src/fetch-docs.py

```diff
@@ -44,6 +47,11 @@
     f'{ncov_url}interpretation.md': 'tutorials/SARS-CoV-2/steps/interpretation.md',
     f'{ncov_url}narratives.md': 'tutorials/SARS-CoV-2/steps/narratives.md',
     f'{ncov_url}naming_clades.md': 'tutorials/SARS-CoV-2/steps/naming_clades.md',
+    f'{nextclade_url}user/nextclade-algorithm.md': 'learn/nextclade.md',
```

Comment:
Was this ([and the link below](https://github.com/nextstrain/docs.nextstrain.org/pull/62/files#diff-f0af9e6baf0df1c1f6c46ad21e9ca0d9bc1e9d35a202b33a0fe3751cfbd89535R16)) trying to include the whole https://docs.nextstrain.org/projects/nextclade/en/latest/user/algorithm/index.html directory/section? I think you need to get all of those files from that directory and then reference the index if so. However, I think you could just link to the nextclade docs in this case.